### PR TITLE
ref(content): Extend swipe down to refresh. Disable home refresh. Dis…

### PR DESCRIPTION
**Pull request by cmgustavo**
…able bounce in all pages

### CODE REVIEW

File: src/pages/settings/key-settings/key-settings.html [hunk 1]
```diff
@@ -1,22 +1,5 @@
...
-<ion-content #scrollArea>
-  <div class="wrapper">
-    <expandable-header [scrollArea]="scrollArea" [fadeFactor]="5" [disableFade]="true" #expandableHeader>
-      <ion-toolbar class="wide-header__title">
-        <expandable-header-primary>
-          {{canSign ? ('Key Settings' | translate) : ('Read Only Wallets' | translate) }}
-        </expandable-header-primary>
-      </ion-toolbar>
-    </expandable-header>
+<wide-header-page title="{{canSign ? ('Key Settings' | translate) : ('Read Only Wallets' | translate) }}">
```
Comment (Gamboster) [inline src/pages/settings/key-settings/key-settings.html:1]: Did you test if the wallet reorder still works correctly by making this change?

Comment (cmgustavo) [reply src/pages/settings/key-settings/key-settings.html:1]: Uhm.. nop. Fixing that now. Thanks!

Comment (cmgustavo) [reply src/pages/settings/key-settings/key-settings.html]: Updated. Reverted changes and added safe-bottom-area manually

**Comment by cmgustavo** [comment]
Updated.

**Pull request merged**